Algorithms/algorithms.py

```diff
@@ -0,0 +1,75 @@
+""" This module contains the Binary Search, Bubble Sort 
+and Recursive Factorial Functions  """
+from typing import Any
+
+def binary_search(arr, target) -> int:
```

Comment:
missing type hint for input function parameters

Reply:
you have added types to dosctring description, but not in function definition itself